test/lib/util.js

```diff
@@ -27,6 +27,14 @@ function concatJson(resolve, reject) {
   };
 }
 
+function delaysJson(ms, json) {
+  return function (req, res) {
+    setTimeout(function () {
+      res.json(json);
+    }, ms);
+  };
+}
+
```

Comment:
this function is only used once, so should not be in reusable util

Reply:
few more tests to add, so it will be shortly